test/helper.lua

```diff
@@ -258,25 +258,27 @@ function helpers.tarantool_version()
     return major, minor, patch
 end
 
-function helpers.vinyl_is_broken()
-    -- Blocked by https://github.com/tarantool/tarantool/issues/6448
+function helpers.vinyl_is_supported()
     local major, minor, patch = helpers.tarantool_version()
 
-    -- Since Tarantool 1.10.11.
-    local broken_v1_10 = major >= 1 and (minor > 10 or minor == 10 and patch >= 11)
-
-    -- Tarantool >= 2.1.0 and < 2.8.2.
-    local broken_v2 = (major >= 2 and major < 3) and (minor >= 1 and minor < 8 or minor == 8 and patch <= 2)
-
-    return broken_v1_10 or broken_v2
+    -- Blocked by https://github.com/tarantool/tarantool/issues/6448
+    -- Tarantool < 1.10.11 for Tarantool v1
+    -- Tarantool <= 2.1.0 or >= 2.10 for Tarantool v2
+    return (major == 1 and minor < 10) or
+        (major == 1 and minor == 1 and patch < 11) or
+        (major == 2 and minor <= 1) or
+        (major == 2 and minor >= 10) or
+        (major >= 3)
 end
```

Comment:
1. I would use `<=`, `>=` consistently, it is easier to interpret as 'starting from X.Y', 'ending at X.Y' -- more intuitive IMHO.
2. `minor == 1` should be `minor == 10` I guess.
3. I'm not sure I understand the comment. Whether the affected range is between introducing `_vinyl_deferred_delete` (1.10.1-309-gf4cb5fc8a) and fixing the linked issue (1.10.11-5-g40d8f4df1, 2.8.2-12-g2663a9c6c, 2.10.0-beta1-51-g0428bbcef)? IOW, supported versions are:
    * 1.10: v <= 1.10.1 and v >= 1.10.12
    * 2.8: v >= 2.8.3
    * 2.10: all
    * 3: all

Reply:
BTW, it does not look as blocked anymore. I would rephrase it.

Reply:
Thank you, it's my fault. I looked at the wrong commit.